core/runtime/block_builder_api.hpp

```diff
@@ -28,7 +28,7 @@ namespace kagome::runtime {
         const primitives::Extrinsic &extrinsic) = 0;
 
     /// Finish the current block.
-    virtual outcome::result<primitives::BlockHeader> finalize_block() = 0;
```

Comment:
actually, both finalize and finalize do exist and mean the same.
probably you have reasons for the change, I don't mind, just wanted to mention it.

Reply:
It's from RE spec.